official/resnet/keras/keras_imagenet_main.py

```diff
@@ -95,6 +96,8 @@ def run(flags_obj):
     raise ValueError('dtype fp16 is not supported in Keras. Use the default '
                      'value(fp32).')
 
+  backend.set_image_data_format(flags_obj.data_format)
```

Comment:
Can we make all these changes to the corresponding Cifar version too? at least to be consistent. I don't know how much performance benefit there will be in that case.

Reply:
Done. Added same logic to Cifar model as well.